code/datums/martial/sleeping_carp.dm

```diff
@@ -124,17 +128,20 @@
 		return FALSE
 	return TRUE
 
-/datum/martial_art/the_sleeping_carp/on_projectile_hit(mob/living/carp_user, obj/projectile/P, def_zone)
-	. = ..()
+/datum/martial_art/the_sleeping_carp/proc/hit_by_projectile(mob/living/carp_user, obj/projectile/P, def_zone)
```

Comment:
can you fix the single-letter var name now that this isn't a subtype of `on_projectile_hit()`